src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -1776,6 +1776,18 @@ private static EmitResult ToEmitResultAndFree(DiagnosticBag diagnostics, bool su
             bool emitPortablePdb = moduleBeingBuilt.EmitOptions.DebugInformationFormat == DebugInformationFormat.PortablePdb;
             string pdbPath = (pdbStreamProvider != null) ? (moduleBeingBuilt.EmitOptions.PdbFilePath ?? FileNameUtilities.ChangeExtension(SourceModule.Name, "pdb")) : null;
 
+            // The PDB path is emitted in it's entirety into the PE.  This makes it impossible to have deterministic
```

Comment:
It could be done around https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs#L437 as @tmat suggests, but I think of this  (i.e. the decision what bits should be placed into the PDB) more as a feature of the core compiler, not the command-line processor. So I like the code here.